### DISCUSSION TITLE
handover support in 7.2 split

### DISCUSSION THREAD

**Discussion by joshuamoorexyz** [srsran/srsRAN_Project#393] [General Help]
Is there any current implementation of handover in the split architecture. If so, what RU is used?

**Comment by andrepuschmann** [comment]
We don't officially support handover yet. Once we do it'll work with any RU.

**Reply by HaoxinSEU** [reply]
Hi @andrepuschmann, 

May I ask when you plan to release the handover code? In the workshop, you revealed that the team had been testing it for some time for stability.

Thanks!

**Reply by andrepuschmann** [reply]
It'll be officially released with 24.4 but I guess we will provide access somewhat earlier along with some tutorials.

**Reply by HaoxinSEU** [reply]
Thanks! Really looking forward to that.
